### ISSUE TITLE
[BUG] [Formatter] Multiline inline content dedents following lines by 1 level (regression in 1.40.8)

### ISSUE THREAD

**Issue by scur-iolus** [djlint/djLint#2298]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint (**1.42.3**)
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: ubuntu 24.04
- Python Version 3.14
- djLint Version 1.42.3
- template language: django

## Issue

The formatter drops one indentation level for everything that follows an inline element which **opens mid-line (after text) and closes on a later line**.

Regression introduced in **1.40.8**, still present in **1.42.3**. 1.40.7 is unaffected.

The output is idempotent, so the wrong indentation is silently persisted on the next run.

## How To Reproduce

MRE:

`input.html`:

```html
<div>
    <p>
        text <b>bold
        more</b> tail
    </p>
    <p>after</p>
</div>
```

```
djlint input.html --reformat
```

**Expected:** file unchanged.

**Actual:** `</p>` and `<p>after</p>` are dedented by one level (`</div>` is untouched only because it is already at column 0):

```html
<div>
    <p>
        text <b>bold
        more</b> tail
</p> <!-- WTF? -->
<p>after</p>
</div>
```

No config file and no `--profile` are needed.

## Scope

- Any inline element triggers it: `<b>`, `<em>`, `<span>`, `<a>`, …
- **Not** triggered when the opening tag starts its own line, that case is handled correctly.
- The loss is exactly one level, no matter how many multi-line inline elements are present.

## Suspected cause

Commit `215ef425e`, *fix(formatter): dedent after trailing inline closing tags*, which fixed #834. In `src/djlint/formatter/indent.py`:

```python
if not is_block_raw and "</" in item and not tag_unindent_pattern.search(item):
    html_depth_change = sum(...)          # net open/close on this line
    dedent_after += max(-html_depth_change, 0)
```

The compensating dedent is applied to every line with a net-negative HTML depth, but the matching indent *increase* only happens when the inline element's opening tag begins its own line.

In #834's shape the two are paired, so the result is correct:

```
<p>
    <a>b
        c</a> d    <- opening <a> indented this line by one level
</p>               <- dedent removes it again
```

Put text before the opening tag and the increase never happens, but the dedent still fires:

```
<p>
    x <a>b
    c</a> d        <- opening <a> did not indent this line
</p>               <- dedent fires anyway, one level is lost
```

At only 2 levels of nesting the loss is invisible because the indent clamps at column 0, which is why the MRE above wraps the `<p>` in a `<div>`.
